small-multiple-line/index.js

```diff
@@ -0,0 +1,281 @@
+ // .subtitle("Put headline |here") //use this if you need to override the defaults
+ .height(1000)
+ .extend('numberOfColumns', 2)
+ .extend('numberOfRows', 4),
+
+    webM: gChartframe.webFrameM(sharedConfig)
+ .margin({ top: 10, left: 10, bottom: 88, right: 5 })
+ // .title("Put headline here")
+ .height(1000)
+ .extend('numberOfColumns', 3)
+ .extend('numberOfRows', 3),
+
+    webL: gChartframe.webFrameL(sharedConfig)
+ .margin({ top: 10, left: 10, bottom: 80, right: 5 })
+ // .title("Put headline here")
+ .height(500)
+ .fullYear(true)
+ .extend('numberOfColumns', 8)
+ .extend('numberOfRows', 1),
+
+    webMDefault: gChartframe.webFrameMDefault(sharedConfig)
+ .margin({ top: 10, left: 10, bottom: 80, right: 5 })
+ // .title("Put headline here")
+ .height(800)
+ .extend('numberOfColumns', 4)
+ .extend('numberOfRows', 2),
+
+    print: gChartframe.printFrame(sharedConfig)
+ .margin({ top: 40, left: 7, bottom: 35, right: 7 })
+  // .title("Put headline here")
+  // .width(53.71)// 1 col
+  .width(112.25)// 2 col
+  // .width(170.8)// 3 col
+  // .width(229.34)// 4 col
+  // .width(287.88)// 5 col
+  // .width(346.43)// 6 col
+  // .width(74)// markets std print
+  .height(150)// markets std print
+  .extend('numberOfColumns', 3)
+  .extend('numberOfRows', 3),
+
+ //    social: gChartframe.socialFrame(sharedConfig)
+ // .margin({ top: 140, left: 50, bottom: 138, right: 40 })
+ // // .title("Put headline here")
+ // .width(612)
+ // .height(612), // 700 is ideal height for Instagram
+
+    video: gChartframe.videoFrame(sharedConfig)
+ .margin({ left: 207, right: 207, bottom: 210, top: 233 })
+ // .title("Put headline here")
+     .extend('numberOfColumns', 4)
+     .extend('numberOfRows', 2),
+};
+
+
+// add the frames to the page...
+d3.selectAll('.framed')
+  .each(function addFrames() {
+      const figure = d3.select(this);
+      figure.select('svg')
+          .call(frame[figure.node().dataset.frame]);
+  });
+parseData.fromCSV(dataFile, dateStructure, { yMin, joinPoints, dataDivisor }).then(({ seriesNames, data, plotData, valueExtent, highlights, annos }) => {
+    Object.keys(frame).forEach((frameName) => {
+        const currentFrame = frame[frameName];
+
+        // define other functions to be called
+        const myYAxis = gAxis.yLinear();// sets up yAxis
+        const myXAxis = gAxis.xDate();// sets up xAxis
+        const myHighlights = lineChart.drawHighlights();// sets up highlight tonal bands
+        const myAnnotations = lineChart.drawAnnotations();// sets up annotations
+        // sets up the legend
+        const myLegend = gLegend.legend(); // eslint-disable-line
+        // const plotDim=currentFrame.dimension()//useful variable to carry the current frame dimensions
+
+        // Create the plot widths, but for each individual graph
+        const widthOfSmallCharts = (currentFrame.dimension().width / currentFrame.numberOfColumns() - currentFrame.rem());
```

Comment:
Group operations using parentheses here. It otherwise throws a linter error and makes the intended expression harder to read.